### DISCUSSION TITLE
Can’t clone at last revision?

### DISCUSSION THREAD

**Discussion by ttamttam** [syl20bnr/spacemacs#16424] [General]
I do not understand why
`git clone https://github.com/syl20bnr/spacemacs.git`
gives me a **8-weeks** old version of develop branch?

Any idea?

**Comment by fnussbaum** [comment]
You might have been looking at the author date of the latest commit 9cddd1543ab19099a53fd98e5f0968138c2306f8, which was merged today?

**Comment by ttamttam** [comment]
Oh! Sorry for the noise. Understood.
